python/cudf/setup.py

```diff
@@ -80,7 +80,6 @@
     ext_modules=cythonize(
         extensions,
         nthreads=nthreads,
-        language="c++",
```

Comment:
We need to specify the language is `c++` somewhere I think. Maybe as a compiler directive?

Reply:
I think this is covered by the `language="c++"` argument to the Extension constructor?

Reply:
I'm not sure. @jakirkham do you happen to know this?

Reply:
Nevermind this looks to be good: https://cython.readthedocs.io/en/latest/src/userguide/wrapping_CPlusPlus.html#specify-c-language-in-setup-py

Reply:
Yeah I think Paul's previous PR ensured we always use C++ in all Cython extensions.